nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -58,6 +58,11 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - `libass` now uses the native CoreText backend on Darwin, which may fix subtitle rendering issues with `mpv`, `ffmpeg`, etc.
 
+- [`lib.customisation.extendMkDerivation`](https://nixos.org/manual/nixpkgs/unstable#function-library-lib.customisation.extendMkDerivation) is introduced to define a build helper supporting [fixed-point arguments](https://nixos.org/manual/nixpkgs/unstable#chap-build-helpers-finalAttrs) (`(finalAttrs: { ... })`) based on an existing one with such support.
```

Comment:
```suggestion
- [`lib.customisation.extendMkDerivation`](https://nixos.org/manual/nixpkgs/unstable#function-library-lib.customisation.extendMkDerivation) is introduced to define a build helpers supporting [fixed-point arguments](https://nixos.org/manual/nixpkgs/unstable#chap-build-helpers-finalAttrs) (`(finalAttrs: { ... })`) based on an existing one with such support.
```

But this should probably not use absolute URLs unless I'm missing something here

Reply:
NixOS manual and Nixpkgs manual are built seperanely. I guess there's no other way for them to refer to each other without using absolute URLs.

Reply:
Oh what, we don't have separate release notes for Nixpkgs and NixOS?! (I couldn't see the whole file path on mobile)